sbt-plugin/src/main/scala/scala/scalajs/sbtplugin/ScalaJSPluginInternal.scala

```diff
@@ -431,8 +431,8 @@ object ScalaJSPluginInternal {
       packageJSDependenciesSetting(packageJSDependencies, "package-js-deps", _.lib),
 
       artifactPath in packageMinifiedJSDependencies :=
-        ((crossTarget in packageJSDependencies).value /
-            ((moduleName in packageJSDependencies).value + "-jsdeps.min.js")),
+        ((crossTarget in packageMinifiedJSDependencies).value /
+            ((moduleName in packageMinifiedJSDependencies).value + "-jsdeps.min.js")),
```

Comment:
Looks like this fixes another (unreported) issue. There should be an issue to reference and fix it in a different commit.